src/main/java/com/google/api/codegen/transformer/java/JavaGapicSurfaceTestTransformer.java

```diff
@@ -87,90 +102,64 @@ public JavaGapicSurfaceTestTransformer(GapicCodePathMapper javaPathMapper) {
     return views;
   }
 
-  private Iterable<Interface> getGrpcInterfacesToMock(Model model, ApiConfig apiConfig) {
-    Map<String, Interface> interfaces = new LinkedHashMap<>();
+  ///////////////////////////////////// Smoke Test ///////////////////////////////////////
```

Comment:
I reorder the sequence of the functions in the file to make it more readable. I think this is the section (Smoke Test) that contains the most new code. Other changes are mostly just moving.